pkgs/development/tools/kafkacat/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, fetchFromGitHub, zlib, perl, pkgconfig, rdkafka, yajl }:
+
+stdenv.mkDerivation rec {
+  name = "kafkacat-${version}";
+  version = "1.3.1";
+
+  src = fetchFromGitHub {
+    owner = "edenhill";
+    repo = "kafkacat";
+    rev = "${version}";
+    sha256 = "1fgs04rclgfwri6vd9lj0mw545nmscav9p6kh7r28k5ap2g0gak5";
+  };
+
+  buildInputs = [ zlib perl pkgconfig rdkafka yajl ];
+
+  configureFlags =  "--includedir=${rdkafka}/include/librdkafka";
```

Comment:
Also, why did I need to add the rdkafka include dir, but yajl's gets added automatically? I can't see a salient difference in the respective expressions (e.g. multiple outputs)

Reply:
You did not need this; I've removed this line.
Both `${rdkafka}/include` and `${yajl}/include` are made available for the build. Since `kafkacat` uses `#include <librdkafka/rdkafka.h>`, this is enough. However, if `kafkacat` used `#include <rdkafka.h>` instead, `${rdkafka}/include/librdkafka` wouldn't be available by default and you'd have to help `kafkacat` find that header with `--includedir=${rdkafka}/include/librdkafka`.

Reply:
Have you tested without? I added this line because it didn't build initially - errors related to an undefined type, and adding this fixed it.

Reply:
But: I don't know why it would work or not, so grain of salt.

Reply:
Certainly I've built it. If this include were necessary, the error would have looked like this:
```
In file included from kafkacat.c:50:0:
kafkacat.h:35:21: fatal error: rdkafka.h: No such file or directory
 #include <rdkafka.h>
                     ^
compilation terminated.
make: *** [mklove/Makefile.base:77: kafkacat.o] Error 1
```